src/modules/__test__/localStorage.test.js

```diff
@@ -0,0 +1,74 @@
+import Storage from '../localStorage.js';
+
+// eslint-disable-next-line func-names
```

Comment:
- [ ] `required`
- [ ] please enable the `eslint` disabling it defeats the purpose of using the linters. 🥂